Microsoft.Identity.Web/WebAppServiceCollectionExtensions.cs

```diff
@@ -23,113 +23,63 @@ namespace Microsoft.Identity.Web
     /// </summary>
     public static class WebAppServiceCollectionExtensions
     {
-        /// <summary>
-        /// Add authentication with Microsoft identity platform.
-        /// This method expects the configuration file will have a section named "AzureAd" with the necessary settings to initialize authentication options.
-        /// </summary>
-        /// <param name="services">Service collection to which to add this authentication scheme</param>
-        /// <param name="configuration">The Configuration object</param>
-        /// <param name="subscribeToOpenIdConnectMiddlewareDiagnosticsEvents">
-        /// Set to true if you want to debug, or just understand the OpenIdConnect events.
-        /// </param>
-        /// <returns></returns>
-        public static IServiceCollection AddMicrosoftIdentityPlatformAuthentication(
-            this IServiceCollection services,
-            IConfiguration configuration,
-            string configSectionName = "AzureAd",
-            bool subscribeToOpenIdConnectMiddlewareDiagnosticsEvents = false)
+        #region
+        [Obsolete("Use AddSignIn")]
```

Comment:
>Use AddSignIn [](start = 19, length = 13)

This method has been deprecated, please use the AddSignIn() method instead.

Reply:
Fixed